README.md

```diff
@@ -234,6 +234,15 @@ You might, for example, include a constraint in your manifest that specifies `ve
 * `0.2.3` becomes the range `>=0.2.3, <0.3.0`
 * `0.0.3` becomes the range `>=0.0.3, <0.1.0`
 
+## Timeouts
+
+`dep ensure` can sometime fail on very slow system due to underlying operations taking too long to perform. This is especially true on Windows due to filesystem issues.
+
+If the built-in timeouts are not suitable to your environment, you can override those by setting specific environment variables:
+
+* `DEP_EXPENSIVE_CMD_TIMEOUT`: Set the maximum number of seconds expensive operations can take. The default is 2 minutes (120 seconds).
```

Comment:
Truth be told, I have no idea how to actually know for sure which operations are performed that happen to timeout.

`dep` verbose mode doesn't tell us exactly what it does and for how long it tries. Perhaps it'd be a good idea if there was a debug flag of some sort that gives that kind of information.

What do you think ?